tools/android/packaging/xbmc/src/XBMCFile.java.in

```diff
@@ -0,0 +1,99 @@
+package @APP_PACKAGE@;
+
+import android.util.Log;
+
+/**
+ * Created by koyin on 07/01/2018.
```

Comment:
typo in your name ;)

Reply:
Yeah, I know. They were created on Android Studio on windows, which truncated my username for reasons only MS know.

I could have put our standard header, but some of those are based on Apache licensed material, so no actual clue which license they have ;)